services/core-api/app/api/mines/mine/resources/mine_basicinfo.py

```diff
@@ -25,6 +26,23 @@ def post(self):
 
         data = self.parser.parse_args()
         mines = data.get('mine_guids', [])
+        simple = data.get('simple', False)
+
+        if simple:
+            mine_table = table(Mine.__tablename__, column('mine_guid'),
+                               column('mine_name'), column('mine_no'),
+                               column('deleted_ind'))
+
+            mines_q = select([mine_table]).where(mine_table.c.deleted_ind == False)
+
+            if mines:
+                mines_q = mines_q.where(mine_table.c.mine_guid.in_(mines))
+
+            connection = db.engine.connect()
+            results = connection.execute(mines_q).fetchall()
+
+            return results
```

Comment:
One thing I noticed here is if mines = [] it will return all mine data instead of empty. Is that true?

Reply:
I don't think so since `mines_q = mines_q.where(mine_table.c.mine_guid.in_(mines))` should dictate finding only the mines who's guids are within the `mines` array.